Changes.md

```diff
@@ -55,6 +70,9 @@ API
 - EditScopeAlgo : Added support for editing attributes.
 - TweakPlug : Added `Create` mode.
 - VectorDataWidget : Added `dataMenuSignal` for constructing the context menu. Classes can add their own slots to this signal to modify the context menu. Derived classes should use it in favor of `_contextMenuDefinition()`.
+- PathListingWidget :
+  - Deprecated `allowMultipleSelection` constructor argument. Use `selectionMode` argument instead.
+  - `setSelection` and `getSelection` can now accept either a single `PathMatcher` that represents the selection for all columns, or a list of `PathMatcher` objects, one for each column. If `selectionMode` is `Row` or `Rows`, `getSelection` returns a single `PatchMatcher` for backwards compatibility.
```

Comment:
This is under the wrong version, and the documentation for `setSelection()` and `getSelection()` doesn't quite reflect our tweaked behaviour for those functions any more.